src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -186,39 +186,41 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.ColonColonToken))
+            if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
             {
-                text = "::_CSharpKeyword";
+                text = "$_CSharpKeyword";
                 return true;
             }
 
-            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
+            if (token.IsVerbatimStringLiteral())
             {
-                text = "cs_namedParameter";
+                text = "@_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.QuestionToken) && token.Parent is ConditionalExpressionSyntax)
+            if (token.IsKind(SyntaxKind.ColonColonToken))
             {
-                text = "?_CSharpKeyword";
+                text = "::_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken))
+            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
             {
-                text = "=>_CSharpKeyword";
+                text = "cs_namedParameter";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.PlusEqualsToken))
+            if (token.IsKind(SyntaxKind.QuestionToken) && token.Parent is ConditionalExpressionSyntax)
             {
-                text = "+=_CSharpKeyword";
+                text = "?_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.MinusEqualsToken))
+            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken))
             {
-                text = "-=_CSharpKeyword";
+                text = "=>_CSharpKeyword";
```

Comment:
This currently matches the comparison operators page and the lambda operators page. This needs a PR into docs to clean up, and a follow up here. I'd suggest removing `=>_CSharpKeyword` from the lambda page, and just using `lambda_CSharpKeyword`.

Tests for both variants would be good too, to ensure we don't regress it in future.

Reply:
@davidwengier The comparison operator page has the following f1 keywords:

```
  - "<_CSharpKeyword"
  - ">_CSharpKeyword"
  - "<=_CSharpKeyword"
  - ">=_CSharpKeyword"
```

none of them is `=>_CSharpKeyword`

Reply:
Oops, you're right! My bad.